app/assets/javascripts/forever_style_guide/nav-fixed.js

```diff
@@ -0,0 +1,19 @@
+$(function () {
+  $('#header-is_fixed .dropdown').on('shown.bs.dropdown', function () {
+    var windowHeight = $(window).height();
+
+    setTimeout(function() { 
```

Comment:
Though the documentation says it should run after all css transitions, effects, etc are complete, in practice 'shown.bs.dropdown' seems to occur in real time w/ the expansion of the dropdown, so the timeout is hacking it to get it to return the height of the header after expansion.